remappings.txt

```diff
@@ -9,4 +9,6 @@ mento-core-2.1.0/=lib/mento-core-2.1.0/contracts/
 mento-core-2.2.0/=lib/mento-core-2.2.0/contracts/
 mento-core-2.3.1/=lib/mento-core-2.3.1/contracts/
 mento-core-2.5.0/=lib/mento-core-2.5.0/contracts/
-merkle-distributor/=lib/merkle-distributor/contracts/
\ No newline at end of file
+merkle-distributor/=lib/merkle-distributor/contracts/
+mento-core-2.6.0-oz/=lib/mento-core-2.6.0/lib/openzeppelin-contracts-next
+mento-core-2.6.0-tp/=lib/mento-core-2.6.0/lib/openzeppelin-contracts-next/contracts/proxy/transparent
```

Comment:
nit: I am not sure about creating remappings for sub dirs like this one

Reply:
Agreed. I did this because the import path was too long without it. The alternative is changing the max line length in the linter + ci